### PR TITLE
implementing autoheight

### PR DESCRIPTION
**Pull request by Xirdion**
This Pull-Requests adds the swiper-`autoheight` parameter.
So you do not have to change the template to set the property for the slider.
It is just a checkbox in the backend to use autoheight in the current slider.

### CODE REVIEW

File: src/Resources/contao/languages/de/tl_content.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderButtons'] = ['Steuerschaltflächen','Zeigt Vor- und Zurück-Schaltflächen an.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPagination'] = ['Nummerierung','Zeigt eine klickbare Nummerierung für die einzelnen Slides an.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPaginationType'] = ['Art der Nummerierung','Die Art der Nummerierung.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Autoheight','Wenn aktiv, passt sich die Höhe des Sliders an die Höhe des aktuell aktiven Elements an.'];
```
Comment (fritzmg) [inline src/Resources/contao/languages/de/tl_content.php]: For the German translation I would call it `Automatische Höhe`.

File: src/Resources/contao/languages/en/tl_content.php [hunk 2]
```diff
@@ -17,6 +17,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderButtons'] = ['Prev/next buttons','Shows the prev/next buttons.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPagination'] = ['Pagination','Shows a clickable pagination to navigate the slides.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPaginationType'] = ['Pagination type','The type of the pagination.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Autoheight','When active, the height of the slider adapts to the height of the currently active element.'];
```
Comment (fritzmg) [inline src/Resources/contao/languages/en/tl_content.php]: For the English translation I think it should be `Auto-height` (with a dash). But you could look at other Contao slider extensions - some of them also have an "auto height" option. I'd translate it the same as they do.

File: src/Resources/contao/languages/de/tl_content.php (hunk 1 above)
Comment (Xirdion) [reply src/Resources/contao/languages/de/tl_content.php]: I have changed the translation.

File: src/Resources/contao/languages/en/tl_content.php (hunk 2 above)
Comment (Xirdion) [reply src/Resources/contao/languages/en/tl_content.php]: I now use `Auto-Height`.
But I did not find another extension with auto-height.

File: src/Resources/contao/languages/en/tl_content.php [hunk 3]
```diff
@@ -17,7 +17,7 @@
 $GLOBALS['TL_LANG']['tl_content']['sliderButtons'] = ['Prev/next buttons','Shows the prev/next buttons.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPagination'] = ['Pagination','Shows a clickable pagination to navigate the slides.'];
 $GLOBALS['TL_LANG']['tl_content']['sliderPaginationType'] = ['Pagination type','The type of the pagination.'];
-$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Autoheight','When active, the height of the slider adapts to the height of the currently active element.'];
+$GLOBALS['TL_LANG']['tl_content']['sliderAutoheight'] = ['Auto-Height','When active, the height of the slider adapts to the height of the currently active element.'];
```
Comment (fritzmg) [inline src/Resources/contao/languages/en/tl_content.php]: Height should be in lower case here I think ;). Also I think you can omit the comma after "When active" (different comma rules for English).

**Comment by Xirdion** [comment]
Translations have been updated.

**Comment by Xirdion** [comment]
Now I hope the translation is okay :)

**Comment by fritzmg** [comment]
Merged and released, thank you :)

**Pull request merged**